src/elements/TableProducerCell/TableProducerCell.js

```diff
@@ -36,7 +36,7 @@ const TableProducerCell = ({ bodyCellClassName, className, producer }) => {
 
   return (
     producer && (
-      <td className={cellClassNames}>
+      <td data-testid={'producer'} className={cellClassNames}>
```

Comment:
better to use here `headerId` instead of hardcoded value 'producer'

Reply:
![Screenshot 2024-03-07 at 1 32 55 p m](https://github.com/pinis-gini-apps/ui/assets/97065270/ba796b19-a85a-4d62-b31f-3145e074cb16)

Reply:
Yes

Reply:
done.